Sources/MapboxDirections/RouteResponse.swift

```diff
@@ -27,15 +27,15 @@ public struct RouteResponse {
 
 extension RouteResponse: Codable {
     enum CodingKeys: String, CodingKey {
-        case code
-        case message
-        case error
+        case code // << do we need these?
+        case message // <<
+        case error // <<
```

Comment:
These values are now decoded using `ResponseDisposition`, but It is only used internally while parsing the response and is not passed to the user. I assume this is what we want, and these keys inside `RouteResponse` are not needed?

Reply:
Still actual

Reply:
I think you’re right, because #418 turns the three fields into an Error in the Result.